data/bL34cHig0.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Habeeb Kareem",
+  "bio": "Aspiring Pentester | IT graduate | Open-Source Enthusiast"
```

You've got a problem here, either commit this change or push from your local environment

```suggestion
  "bio": "Aspiring Pentester | IT graduate | Open-Source Enthusiast",
```

---

@Dun-sin, thanks. I committed the change.